CONTRIBUTING.md

```diff
@@ -14,6 +14,12 @@ Here are questions you can answer before you file a bug to make sure you're not
 
 GitHub supports [markdown](https://help.github.com/articles/github-flavored-markdown/), so when filing bugs make sure you check the formatting before clicking submit.
 
+## Contributor License Agreement ("CLA")
+
+In order to accept your pull request, we need you to submit a CLA. You only need to do this once to work on any of Winton's open source projects.
+
+The text of the CLA can be seen in this gist: [Winton CLA](https://gist.github.com/winton-admin/0af4506b3be40fd1d9f3bf27ea4391d2)
```

Comment:
Maybe link directly to https://cla-assistant.io/wintoncode/winton-kafka-streams?

Reply:
Done, thanks.